net/axel/files/patch-axel-tiger-no-posix-c-source.diff

```diff
@@ -1,41 +0,0 @@
-Tiger does not have the _DARWIN_C_SOURCE override to enable chunks of headers
-when _POSIX_C_SOURCE is defined, so we have to specifically remove the
-_POSIX_C_SOURCE from these headers and files for Tiger builds to succeed.
-
-This patch could not be sent upstream as is... and probably would not be accepted
-if it were sent upstream given the age of Tiger.
-
-kencu@macports.org
```

Comment:
Since Ken said he could not send this patch upstream as is, did you verify whether this patch was still needed to build on Tiger? If not, I would assume it's still needed and would keep it.

Reply:
The patch is for a problem that has an upstream fix, so it shouldn't be needed anymore imo. Plus it causes patch errors in this version cuz target files were changed.

Reply:
...actually seems one file does still require patching. Will make a patch later

Reply:
I have now booted up an older Mac with Tiger and verified that axel 2.17.6 with the patch builds fine as expected and that when applying your changes from this PR axel 2.17.8 without the patch does not build.

As far as I can see all they did to fix the upstream issue was to define `_DARWIN_C_SOURCE` so certainly the part of the portfile that defined that can be removed as that would be redundant. But as the comment you removed from the portfile says, "Tiger needs a different workaround for the same issue" so that patch is still needed. If it no longer applies, then it needs to be reworked so that it does apply.

I've committed an updated version of the patch that works for me. These two commits should be squashed into one when this PR is merged.

Reply:
Thanks for the patch, just found the same thing by reading their source. Should I just squash them on my end?

Reply:
If you like, or the "Squash and merge" button in the web interface can be used.